Doc/library/urllib.request.rst

```diff
@@ -657,6 +662,10 @@ OpenerDirector Objects
    timeout setting will be used). The timeout feature actually works only for
    HTTP, HTTPS and FTP connections).
 
+   .. note::
+
+      The global default timeout value is to never time out, which is often
```

Comment:
I believe that this is the original wording and should be updated to the revised wording (as above). Note also [this pr](https://github.com/python/cpython/pull/17843) which adds a similar note into `http.client`, so there may be some conflict here if that's merged.

Reply:
You are right, thanks for spotting this - I forgot to pull changes when making a previous update.

The purpose of this PR is to point out what the global default timeout is, and that it's often not a good idea to use that value. The #17843 doesn't clearly point out these 2 things so I think if it's merged, it shouldn't have effect on this PR.